### PR TITLE
fixed regression analyse for scylla enterprise

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: sdcm/results_analyze.py [hunk 1]
```diff
@@ -116,12 +116,14 @@ def _test_stats(self, test_doc):
...
+        if test_doc['_source'].get('versions'):
+            for v in ('scylla-server', 'scylla-enterprise-server'):
+                k = test_doc['_source']['versions'].get(v)
+                if k:
+                    return k
 
-        return (test_doc['_source']['versions']['scylla-server']['version'],
-                test_doc['_source']['versions']['scylla-server']['date'])
+        logger.error('Scylla version is not found for test %s', test_doc['_id'])
+        return None
```
Comment (bentsi) [inline sdcm/results_analyze.py]: Python returns None by default

File: sdcm/results_analyze.py [hunk 2]
```diff
@@ -258,7 +262,7 @@ def check_regression(self, test_id, is_gce=False):
         # send results by email
         results = dict(test_type=test_type,
                        test_id=test_id,
-                       test_version=doc['_source']['versions']['scylla-server'],
+                       test_version=test_version_info,
                        res_list=res_list)
         logger.info('Regression analysis:')
         logger.info(pp.pformat(results))
```
Comment (bentsi) [inline sdcm/results_analyze.py]: it can be very big list, change log level to debug

File: sdcm/results_analyze.py (hunk 1 above)
Comment (larisau) [reply sdcm/results_analyze.py]: If method returns value in a normal case, I prefer return explicitly in a bad case.

File: sdcm/results_analyze.py (hunk 2 above)
Comment (larisau) [reply sdcm/results_analyze.py]: changed

**Comment by larisau** [comment]
ok, I'll change it

**Pull request merged**